CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## 0.5.0 - 2020-12-01
+
+### Added
+
+- Publish local projects directly to GitHub
+- Advanced creation to have more control over creation
+- Download remote app settings
+- Notification for when long running GitHub actions complete
+
+### Changed
+- Functions/App Setting tree items no longer appear if build doesn't contain functions
+- `Create New Static Web App` prompts now asks for deployment method
+- Completed GitHub actions cannot be cancelled, on-going actions cannot be re-run
```

Comment:
Should this go in the "Fixed" section? It seems like its more of a bug 🐛

Reply:
Hmm, I think that this was more of a by-design choice originally.  We wanted to increase discoverability by having the node there, even if it led to an error.  It's more of a change now that we include the learn more node.